katalog/etcd-backup-s3/MAINTENANCE.md

```diff
@@ -0,0 +1,5 @@
+# ETCD Backup S3 Maintenance
+In order to update and maintain the package:
+- First check if new images are available for `etcd` and `rclone`
```

Comment:
the version of etcd can be always latest? or it should be aligned with the version of etcd that is running?

Reply:
The considerations about this are the same of the other comment, we're using the `etcd` image only to run `etcdctl snapshot save`. https://github.com/sighupio/fury-kubernetes-dr/pull/80#discussion_r1987008879